test_package/test_package.cpp

```diff
@@ -14,14 +14,13 @@ int main() {
   static_assert("structvar" == name);
 
   // Nameof.
-  using std::literals::string_view_literals;
-
   std::string_view res1 = NAMEOF(structvar);
   std::string_view res2 = NAMEOF(::structvar);
 
   std::cout << res1 << '\n' << res2 << '\n';
+  using namespace std::literals::string_view_literals;
 
-  bool success = res1 == "structvar" sv && res2 == "structvar" sv;
+  bool success = (res1 == "structvar"sv) && (res2 == "structvar"sv);
```

Comment:
There is no need to use parenthesis, but probably you are right and it is more readable in this way. So be it.